src/components/Greetings.js

```diff
@@ -0,0 +1,17 @@
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { randomGreeting } from '../redux/greeting';
+
+const Greetings = () => {
+  const greetings = useSelector((state) => state.greeting);
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(randomGreeting());
+  }, []);
```

Comment:
Kindly include a dependency `dispatch` in the UseEffect hook. This will complete your useEffect and make it trigger to return the result.